README.md

```diff
@@ -33,10 +31,25 @@ Clone the repository
 ```
 $ git clone https://github.com/fossasia/loklak_wok_android.git 
 ```
-Once the project is cloned successfully, import it as a **gradle project**.
-```
-File > New > Import Project > Locate build.gradle file in the cloned directory > Click OK
-```
+Before you begin, you should already have the Android Studio SDK downloaded and set up correctly. You can find a guide on how to do this here: [Setting up Android Studio](http://developer.android.com/sdk/installing/index.html?pkg=studio)
+
+### Setting up the Android Project
+
+1. Download the _loklak_wok_android_ project source. You can do this either by forking and cloning the repository (recommended if you plan on pushing changes) or by downloading it as a ZIP file and extracting it.
```

Comment:
You can make it as `[`loklak_wok_android` project source](https://github.com/fossasia/loklak_wok_android)` so that users can readily find the repo from other locations where this README is rendered.